go/app/main.go

```diff
@@ -129,15 +167,16 @@ func getImg(c echo.Context) error {
 }
 
 func getAllItems(c echo.Context) error {
-	items, err := dataJson()
+	prepareDB()
+	items, err := dbData()
 	if err != nil {
 		fmt.Println(err)
 	}
 	return c.JSON(http.StatusOK, items)
 }
 
 func getItem(c echo.Context) error {
-	items, err := dataJson()
+	items, err := dbData()
```

Comment:
OPTIONAL:
In json, there is no way to fetch specific item_id record, however, you can fetch specific record by SQL.
If you have time, could you try get only item_id specified, instead of fetching all data?
You can do it with another PR and only when you have time (maybe finish other steps)